src/backend/core/resource_server_api/serializers.py

```diff
@@ -0,0 +1,46 @@
+"""Client serializers for the People core app resource server API."""
+
+from rest_framework import serializers
```

Comment:
I was wondering what is the "cleanest" way for this. Maybe it's only the naming that refrains me to do so, how should be named the "classic" API module? I don't want to name it `api` because it would result in `core.api.api` ^^' 
Actually, I'm wondering if the whole "resource_server" part should not be moved outside the "core" module?

If you have something in mind, I would happily discuss it with you :)